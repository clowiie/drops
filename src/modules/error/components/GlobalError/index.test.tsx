import React from 'react'

import { render } from '@testing-library/react'

import * as GlobalErrorType from '.'

describe('GlobalError', () => {
  const { default: GlobalError } = require('.') as typeof GlobalErrorType

  describe('snapshots', () => {
    it('should match snapshots', () => {
      const { container } = render(<GlobalError />)

      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
