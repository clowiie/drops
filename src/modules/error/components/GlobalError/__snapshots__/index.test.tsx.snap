// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalError snapshots should match snapshots 1`] = `
<span
  class="text-2xl font-semibold text-center"
>
  Something went wrong
</span>
`;
